python/graphstorm/config/argument.py

```diff
@@ -457,6 +459,15 @@ def verbose(self):
 
         return False
 
+    @property
+    def logging_level(self):
```

Comment:
Shall we add a --logging-level to allow user set logging level through arugment.

Reply:
i did